README.md

```diff
@@ -1,14 +1,23 @@
-Backdrop Contributed Project Group Application
+What is the difference between Backdrop Contrib Project Group and backdrop-ops/contrib?
 ==============================================
 
-Interested in joining the growing Backdrop contributor community at
-https://github.com/backdrop-contrib?
+The Backdrop Contrib Project Group is an organization structure that retains ownership of the individual project repositories as developers come and go from projects.  Managing projects within the Backdrop Contrib Project Group facilitates the packaging and discovery of projects on Backdropcms.org, collaboration with other member of the Backdrop Contrib Project Group and security coverage. 
 
-To apply simply create a request as an issue at
-https://github.com/backdrop-ops/contrib/issues/new.
+The backdrop-ops/contrib/issues can be used to get feedback and find collaborators for a potential Backdrop project and to apply to join the growing community of developers maintaining a module or theme hosted at https://github.com/backdrop-contrib/ and discuss [META] issues realted to contributing to Backdrop.
```

Comment:
"The backdrop-ops/contrib/issues can be used" ...needs to be clarified. I get that you are pointing to the issue queue so maybe "The Backdrop contrib issue queue (link) can be..."

Reply:
also a typo in "issues realted to contributing to Backdrop." change to "related".